.gitignore

```diff
@@ -11,7 +11,7 @@ bin/
 obj/
 src/packages/
 src/TestResults/
-src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Actual/
+src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Actual/
```

Comment:
>BaselineTestData [](http://example.com/codeflow?start=40&length=16)

Ouch, VS by default ignored this file and I missed it.
way to improve my search skills.